sdk/management/README.md

```diff
@@ -70,7 +70,7 @@ In addition, Azure subscription ID can be configured via environment variable `A
 With above configuration, `azure` client can be authenticated by following code:
 
 ```java
-AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE, true);
+AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
```

Comment:
Currently, we don't have a AzureProfile constructor with only one parameter.

Reply:
That is a pending discussion with Chuang. `true` or `false` here is not very human friendly. So a proposal is either take a `Configuration` object, or just remove it (since there is another constructor that takes `tenantId` `subscriptionId` `azureEnvironment`).